collect_app/src/main/java/org/odk/collect/android/database/FormDatabaseMigrator.java

```diff
@@ -223,6 +225,17 @@ private void upgradeToVersion9(SQLiteDatabase db) {
         SQLiteUtils.dropTable(db, temporaryTable);
     }
 
+    private void upgradeToVersion10(SQLiteDatabase db) {
+        String temporaryTable = FORMS_TABLE_NAME + "_tmp";
```

Comment:
This migration won't actually carry over the state from the old `deleted` column. My assumption is that it's ok for beta users to have their form "restored" if they'd already soft deleted it when they install the update.

Reply:
Yes, fine by me. @opendatakit/testers note that this is expected.